rslib/src/deckconfig/update.rs

```diff
@@ -83,15 +88,18 @@ impl Collection {
 
     fn get_current_deck_for_update(&mut self, deck: DeckId) -> Result<CurrentDeck> {
         let deck = self.get_deck(deck)?.ok_or(AnkiError::NotFound)?;
+        let normal = deck.normal()?;
+        let today = self.timing_today()?.days_elapsed;
 
         Ok(CurrentDeck {
             name: deck.human_name(),
-            config_id: deck.normal()?.config_id,
+            config_id: normal.config_id,
             parent_config_ids: self
                 .parent_config_ids(&deck)?
                 .into_iter()
                 .map(Into::into)
                 .collect(),
+            limits: Some(normal.to_limits(today)),
```

Comment:
One minor downside of doing it this way is that we don't remember what the previously set 'today' limit was after the day rolls over, so if the user frequently extends the limits, they'll have to enter in the desired value each time. We could achieve it by passing the configured limit regardless of day, along with a bool or enum indicating whether it happened to be active at the time. Whether that's worth bothering with at the moment though, I don't know - wouldn't be too hard to add later if we see enough demand for it. If we do add it in the future, we'd probably want to preserve the limits in schema11 as well, or downgrade-and-quit would forget the defaults.

Reply:
Yes, I thought about the same thing, but took the easy path. If you think it's worth it, I'll take care of it.

Reply:
Who'd have thought the trickiest part would be mapping day limits between the frontend and backend representations? But I think everything works as expected now.

Reply:
Thanks for looking into that :-)